frontend/src/libs/helpers/get-valid-class-names.ts

```diff
@@ -0,0 +1,6 @@
+import clsx from 'clsx';
+
+const getValidClassNames = (newStyle?: string, defaultStyle?: string): string =>
+  clsx(defaultStyle, newStyle);
```

Comment:
What if we have more than 2 styles? Consider this possibility as well

Reply:
Also we may want to use not only strings but also conditions, so please use something like 
```
const getValidClassNames = (...inputs: ClassValue[]): string => clsx(...inputs);
```